src/main/java/io/github/minecraftcursedlegacy/api/registry/Registry.java

```diff
@@ -248,4 +246,44 @@ protected void onRemap(T remappedValue, int newSerialisedId) {
 	public static void lockAll() {
 		RegistryRemapper.registries().forEach(r -> r.locked = true);
 	}
+
+	/**
+	 *
+	 * @return all objects stored in this registry.
+	 */
+	public Set<T> values() {
+		return this.byRegistryId.values();
+	}
+
+	/**
+	 *
+	 * @return the ids of all objects stored in this registry.
+	 */
+	public Set<Id> ids() {
+		return this.byRegistryId.keySet();
+	}
+
+	/**
+	 *
+	 * @return the serialised ids of all objects stored in this registry
```

Comment:
Can we have a full stop at the end of this sentence?

Reply:
Fixed in 3fa96f1.